common-lib/src/main/scala/com/gu/mediaservice/model/UsageRights.scala

```diff
@@ -302,3 +304,18 @@ object Pool {
  implicit val jsonReads: Reads[Pool] = Json.reads[Pool]
  implicit val jsonWrites: Writes[Pool] = UsageRights.defaultWrites
 }
+
+
+case class CrownCopyright(restrictions: Option[String] = None)
```

Comment:
I thought we could have something like:
```
Copyright(copyright: String) {
  def cost = copyright match {
    case "crown-copyright" => Free
    case _ => Pay
  }
}
```

But it starts to feel hierarchical i.e. `UsageRights :> Copyright :> CopyrightType`